challenges.yml

```
@@ -2088,3 +2088,217 @@ extending_prop_transition:
           time the value of `key` changes. This means that we now have effectively 2
           components to transition over!
       correct: d
+
+extending_checkout_form:
+  question: |
+    In summary, what does v-model do to an input element when we add to it?
+  explanation: |
+    Essentially, what `v-model` is keep the data in synk between the UI and the
+    data. Soon we'll learn *exactly* how Vue does this!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          `v-model` creates a model of an input element that corresponds to an object.
+          In this object we can set styles, properties and the input's value.
+        b: |
+          `v-model` triggers events at certain points of the application life cycle so
+          that an input element can load a piece of data.
+        c: |
+          `v-model` ensures that our input element stays in sync with the data element we
+          pass to it.
+        d: |
+          `v-model` triggers keyboard events at all times so that we can update our
+          data according to an input element's value.
+      correct: c
+
+extending_checkout_form2:
+  question: |
+    `v-model` has some "modifiers" we can add to it that change the way
+    we process data as it is entered into an input element.
+
+    Which of these options is *not* a `v-model` feature that we've seen in our
```

```suggestion
    Which of these options is *not* a valid `v-model` modifier?
```

I was confused by the "seen in our tutorials" - I was thinking the question was about "which of these valid features did we show vs did we not show"